controllers/logging_controller.go

```diff
@@ -146,6 +148,23 @@ func SetupLoggingWithManager(mgr ctrl.Manager, logger logr.Logger) *ctrl.Builder
 				logger.Error(err, "failed to list logging resources")
 				return nil
 			}
+			if o, ok := object.(*corev1.Secret); ok {
+				requestList := []reconcile.Request{}
+				for key, _ := range o.Annotations {
+					r := regexp.MustCompile("logging.banzaicloud.io/(.*)")
+					result := r.FindStringSubmatch(key)
+					if len(result) > 1 {
+						loggingRef := result[1]
+						// When loggingRef is default we trigger "empty" and default loggingRef as well
```

Comment:
```suggestion
						// When loggingRef is default we trigger "empty" and default loggingRef as well, because we can't use an empty string in the annotation, thus we refer to `default` in case the loggingRef is empty
```